core/domain/user_query_jobs_one_off.py

```diff
@@ -45,17 +45,20 @@ class UserQueryOneOffJob(jobs.BaseMapReduceOneOffJobManager):
     @staticmethod
     def _is_user_inactivity_query_satisfied(user_settings_model, query_model):
         """Determines whether a user has been inactive for n days."""
+        last_active_date = datetime.datetime.min
```

Comment:
I actually found the previous approach more readable--I don't understand the need for ``max()`` below since we're only initializing this one time. Why the change?

Reply:
A user might have created an exploration a long time back but he/she might be editing explorations regularly. This doesn't mean that the user is inactive. So I have chosen the maximum of the two time.
In the previous implementation only one of the two times were considered.

Reply:
That's an actual behavior change that should be documented clearly here. Have tests also been updated to reflect the new behaviors here? There must be some cases that will now behave differently since the logic has changed.

Reply:
There are no test cases that behave differently. I have additional tests. Done.

Reply:
Thanks!